_posts/2025-01-29-sdg23-writer.md

```diff
@@ -0,0 +1,29 @@
+title="NumFOCUS Foundation" alt="NumFOCUS Foundation"
+style="float: right; width: 10em;" />
+
+Thanks to the support of a [Small Development Grant](https://numfocus.org/programs/small-development-grants) awarded to MDAnalysis by [NumFOCUS](https://numfocus.org/), MDAnalysis is excited to start working with Namir Oues (@namiroues) as the technical writer for the funded project, "Unified and comprehensive documentation and learning resources for MDAnalysis".
+
+MDAnalysis has an ever-expanding international community of users and developers. Thorough documentation and extensive learning resources -- such as tutorials and online workshop materials -- are publicly available to users under open source licenses; however, a considerable amount of overlap between the many resources (e.g., [website]({{ site.baseurl }}), [user guide]({{ site.docs.userguide }}), [docs]({{ site.docs.mdanalysis.url }}), [GitHub wiki]({{ site.github.wiki }})) can make it somewhat difficult for self-learners to find the information they need.
+
+As Technical Writer, [Namir](#meet-namir-oues) is tasked with consolidating the main website and additional MDAnalysis learning resources to guide users through a more streamlined workflow by removing duplication and outdated materials and more clearly delineating between developer-focused and user-focused content. A  primary objective of this project is to enhance the user experience for newcomers to MDAnalysis and encourage continued growth in our user and developer communities.
+
+### Meet Namir Oues
+
+<img
+src="Picture Here"
+title="Namir Oues" alt="Namir Oues"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+Please write a short (1-2 paragraphs) bio here. Feel free to include your affiliation(s) and any personal and/or professional interests. Please include any contact information you feel comfortable sharing publicly (e.g., GitHub/Discord handle, LinkedIn, website, blog, etc.)
+
+### Follow Along with this Project
+
+We encourage you to see how this work is progressing and to share your thoughts and feedback. There are many ways to get involved, including joining the [`developers`](https://discord.com/channels/807348386012987462/808088023957897258) and [`documentation`](https://discord.com/channels/807348386012987462/1021430544543785021) channels on our [Discord server]({{ site.discord.url }}) (join using [this invite link]({{ site.discord.invite }})), engaging with the GitHub issue trackers for the [main MDAnalysis repo]({{ site.github.repo }}) and [MDAnalysis User Guide repo](https://github.com/MDAnalysis/UserGuide), and interacting on the [{{ site.mailinglists.discussion.name}} forum]({{ site.mailinglists.discussion.url }}). Ultimately, the goal of this project is to meet the needs of our community, so please do chime in with your input!
+
+— @IAlibay @jennaswa @lilyminium @micaela-matta @orbeckst (Co-PIs/Project Personnel) @namiroues (Technical Writer)
```

Comment:
Change the PI order, should be clear who's the PI (@jennaswa , right?) and roughly in which order people are taking on responsibility, i.e., @micaela-matta needs to come second; for the remainder (@IAlibay, @lilyminium , and I) I chose alphabetical order. I am not sure if there's any specific title attached to the named key personnel so I left that out. If anyone wants to put "co-PI" or "co-I" or whatever there please say something.

I'd suggest
```suggestion
- @namiroues (Technical Writer)
- @jennaswa (PI), @micaela-matta @IAlibay @lilyminium @orbeckst 
```